src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+    private String color;
+    private String type;
+
```

Comment:
The class should not start with an empty line. Please remove the redundant empty line at the beginning of the class definition.